### DISCUSSION TITLE
MSSQLToGCSOperator

### DISCUSSION THREAD

**Discussion by benitodelacruz** [apache/airflow#28132] [Q&A]
using MSSQLToGCSOperator (3.2.0) does not create empty file, inside bucket,  when query return no data. When upgrade to airflow 2.3.3.
Dag works fine when data query is returned.
...
from airflow.providers.google.cloud.transfers.mssql_to_gcs import MSSQLToGCSOperator
  mssql_to_gcs = MSSQLToGCSOperator(
        task_id='MYSQL_TO_GCS_{0}'.format(TABLE_NAME),
        mssql_conn_id='con_mssql_dba_prd',
        gcp_conn_id='google_cloud_storage_default',    
        sql='select_{0}.sql'.format(TABLE_NAME),
        bucket=SOURCE_BUCKET,
        filename='composer/{0}/gcp_{0}{1}.json'.format(TABLE_NAME, DATE_FORMAT)
        dag=dag
     )
...

**Comment by potiuk** [comment]
Create an issue (or better fix it via PR) then.
